Week-2/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Week 2 Assignment 1 - Loren Ipsum</title>
```

Comment:
@rajeshvlega9 Can you create two branches for these two assignments

Reply:
i will create it from next time. thank you.